core/model/MqttMessage.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2021 WolkAbout Technology s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MESSAGE_H
+#define MESSAGE_H
+
+#include <string>
+
+namespace wolkabout
+{
+/**
+ * This class represents a raw message that is inbound/outbound over MQTT.
+ * The message is composed of the topic that the message was sent over or received over,
+ * and the message content that was received/sent out.
+ */
+class MqttMessage
```

Comment:
The `Message` name itself became really ambiguous in the project, considering we have a bunch of `****Message` classes and none of them have to do with the `Message` class itself, rather with the `MessageObject` class. Since both of them are messages, but none of them is `"the message"` I thought this would make it more clear, but I will revert the change.